src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -62,6 +63,7 @@ public void generatePage() {
 
     private void generateReport() {
         context.put("report_file", getWebPage());
+        context.put("escHtml", StringEscapeUtils.ESCAPE_HTML4);
```

Comment:
From the documentation of [StringEscapeUtils.html#escapeHtml4](https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/StringEscapeUtils.html#escapeHtml4(java.lang.String)): 
_Supports all known HTML 4.0 entities, including funky accents. Note that the commonly used apostrophe escape character (&apos;) is not a legal entity and so is not supported)._ 

And from the [Wikipedia entry on HTML entities](https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references#Character_entity_references_in_HTML) we can see that none have been added for HTML5. Footnotes q and r are interesting but shouldn't be an issue for any modern browser.